website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -88,34 +88,26 @@ instance, and click **Authenticate**.
    instance's Global scope via the web UI. This is the default scope for all new
    Boundary clusters.
 
-1. Click **Auth Methods**.
-
-1. Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
-authenticate to HCP Boundary using the CLI.
-
 <Tabs>
 <Tab heading="CLI">
 
-
 1. In the HCP Boundary portal, open the **Boundary Overview** page, and click
 the **copy icon** in the **Copy this into Boundary Desktop** section. This
 copies your environment’s Boundary **Cluster URL**.
 
 1. Open a terminal session and set the `BOUNDARY_ADDR` environment variable to
 the copied Cluster URL.
 
-```shell-session
-$ export BOUNDARY_ADDR=<boundary-cluster-url>
-```
+    ```shell-session
+    $ export BOUNDARY_ADDR=<boundary-cluster-url>
+    ```
 
 1. Log in with the administrator credentials you created when you deployed the
 HCP Boundary instance.
 
-```shell-session
-$ boundary authenticate password \
-   -auth-method-id=<boundary-auth-method-id> \
-   -login-name=<boundary-admin-username>
-```
+    ```shell-session
+    $ boundary authenticate
+    ```
```

Comment:
also rendering incorrectly in the deploy preview, likely because of indentation. I'd also like to add in the output, so users know what to expect when they run the command

```suggestion
```shell-session
$ boundary authenticate
Please enter the login name (it will be hidden):
Please enter the password (it will be hidden):

Authentication information:
  Account ID:      acctpw_VOeNSFX8pQ
  Auth Method ID:  ampw_wxzojlKJLN
  Expiration Time: Mon, 13 Feb 2023 12:35:32 MST
  User ID:         u_1vUkf5fPs9

The token was successfully stored in the chosen keyring and is not displayed here.
```
```